### ISSUE TITLE
small electric and large electric generators are more efficient then car alternator

### ISSUE THREAD

**Issue by Phenomphear** [Noctifer-de-Mortem/nocts_cata_mod#62]
![image](https://user-images.githubusercontent.com/28910998/49741297-58f90500-fc53-11e8-8365-d79d49a59b16.png)

**Comment by mlangsdorf** [comment]
Yeah, please update all your vehicle_part power (but not epower) amounts by x373.

**Comment by chaosvolt** [comment]
Welp, these are vehicle parts I didn't even know existed in Cata++, guess I'll look into getting a fix up if Noct isn't able to today.
